sqsdomain/pools.go

```diff
@@ -149,9 +149,9 @@ func (p *PoolWrapper) Validate(minPoolLiquidityCapitalization osmomath.Int) erro
 
 	// Validate pool liquidity capitalization.
 	// If there is no pool liquidity capitalization error set and the pool liquidity capitalization is nil or zero, return an error. This implies
-	// That pool has no liqudiity.
+	// That pool has no liquidity.
 	poolLiquidityCapError := strings.TrimSpace(p.SQSModel.PoolLiquidityCapError)
-	if poolLiquidityCapError == "" && sqsModel.PoolLiquidityCap.IsZero() {
+	if poolLiquidityCapError == "" && (sqsModel.PoolLiquidityCap.IsNil() || sqsModel.PoolLiquidityCap.IsZero()) {
```

Comment:
Thanks. Do you mind adding a quick changelog entry in CHANGELOG.md?

Reply:
Thanks, I have added an entry in a change log.